src/functions-and-arrays.js

```diff
@@ -52,15 +120,34 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+function uniquifyArray(uniquiFy) {
+  if (uniquiFy.length === 0){
+    return null
+  }
+  let anotherUniquify = [];
+  for (let i = 0; i < uniquiFy.length; i++){
+    if (anotherUniquify.indexOf(uniquiFy[i]) === -1){ //Colocamos ("=== -1") porque estamos a usar um index que não existe
+      anotherUniquify.push(uniquiFy[i])
+    }
+  }
+  return anotherUniquify
+}
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
-
+function doesWordExist(wordExist2, wordSearch) {
+  if (wordExist2.length === 0){
```

Comment:
Nice function, though it is easier to read the code if you name the array and the word like this:

`function doesWordExist(wordsArr, word) {`